pkg/klusterlet/clusterclaim/clusterclaimer.go

```diff
@@ -255,6 +291,15 @@ const (
 	OCP3Version = "3"
 )
 
+func getOCPID(configv1Client openshiftclientset.Interface) (clusterID string, err error) {
```

Comment:
we already have a method to get clusterID `version, clusterID, err := c.getOCPVersion()` ,and the claim name is ClaimOpenshiftID.

Reply:
Yes,the design current is using openshiftID -> namespace uid -> randomID, but I think finally we should come up with a consist way to generate this ID.